File: concert-ticketing/src/main/java/com/hhplus/concertticketing/ConcertTicketingApplication.java
package com.hhplus.concertticketing;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@EnableCaching
@SpringBootApplication
public class ConcertTicketingApplication {

    public static void main(String[] args) {
        SpringApplication.run(ConcertTicketingApplication.class, args);
    }

}

File: concert-ticketing/src/main/java/com/hhplus/concertticketing/ConcertTicketingApplication.kt
package com.hhplus.concertticketing

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

// 메인 애플리케이션 클래스
@EnableScheduling
@EnableCaching
@SpringBootApplication
class ConcertTicketingApplication

fun main(args: Array<String>) {
    // 스프링 부트를 실행하는 진입점
    runApplication<ConcertTicketingApplication>(*args)
}
